app/controllers/coronavirus_form/living_with_vulnerable_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::LivingWithVulnerableController < ApplicationController
+  def submit
+    @form_responses = {
+      living_with_vulnerable: strip_tags(params[:living_with_vulnerable]).presence,
+    }
+
+    invalid_fields = validate_radio_field(
+      controller_name,
+      radio: @form_responses[:living_with_vulnerable],
+    )
+
+    if invalid_fields.any?
+      flash.now[:validation] = invalid_fields
+      log_validation_error(invalid_fields)
+      render controller_path
+    elsif session["check_answers_seen"]
```

Comment:
I don't think we need this `elsif`, since we don't have a check answers page.

Reply:
👍